Apps/RaptureIntTests/src/test/java/rapture/structured/StructuredApiIntegrationTests.java

```diff
@@ -86,7 +87,7 @@
      */
     @BeforeClass(groups = { "nightly", "search" })
     @Parameters({ "RaptureURL", "RaptureUser", "RapturePassword" })
-    public void setUp(@Optional("http://localhost:8665/rapture") String url, @Optional("rapture") String username, @Optional("rapture") String password) {
+    public void setUp(@Optional("http://192.168.99.100:8665/rapture") String url, @Optional("rapture") String username, @Optional("rapture") String password) {
```

Comment:
looking at the setups/beforeTests for the other integration tests they have localhost set
not 100% clear on how that works since we specify the target in an xml